### PR TITLE
More Footer changes

### PR DESCRIPTION
**Pull request by RoyZone96**
Footer Css added

### CODE REVIEW

**Review by Valtaire-O — COMMENTED** [review]
Lgtm

File: src/components/Footer/Footer.css
Comment (markdreed512) [inline src/components/Footer/Footer.css:1]: Looks like there's still substantial differences here between this styling and the figma design requirements.  I'm seeing differences in font color, margin, padding, background color (remember that shade of purple is stored in a css variable `--secondary-purple` and all site-wide variables can be referenced in App.css). And also no C&C logo.

Comment (RoyZone96) [reply src/components/Footer/Footer.css:1]: I dont have a png for the logo on hand if you can give me one i can include it

Comment (Valtaire-O) [reply src/components/Footer/Footer.css:1]: I put them in slack https://baltimoretech.slack.com/archives/C07M5CLJ677/p1729094360917879?thread_ts=1729094273.936549&cid=C07M5CLJ677

**Pull request closed without merging**